socorro/tests/external/test_crashstorage_base.py

```diff
@@ -150,204 +145,6 @@ def test_basic_crashstorage(self):
 
             crashstorage.close()
 
-    def test_polyerror(self):
```

Comment:
Instead of having a poly-storage thing, we're going to move to a "processing supports multiple destinations" model. It's much easier to implement, it doesn't require additional poly-storage code, and it removes the weirdness around polystorageerror which is hard to debug in the logs and in sentry.